packages/imperative/src/rest/index.ts

```diff
@@ -17,11 +17,11 @@ export * from "./src/client/doc/IRestClientError";
 export * from "./src/client/doc/IRestClientResponse";
 export * from "./src/client/doc/IRestOptions";
 export * from "./src/client/Headers";
-export * from "./src/client/AbstractRestClient";
-export * from "./src/client/CompressionUtils";
+// export * from "./src/client/AbstractRestClient";
```

Comment:
While none of our apps directly import AbstractRestClient, we recently discovered that a member of the external Zowe community derives their own class by inheriting directly from AbstractRestClient. Until we implement a means for consumers to specify credential order-of-precedence, I think we will need to keep AbstractRestClient available to consumers.